src/components/uikit/inputs/TextInput.tsx

```diff
@@ -65,25 +83,48 @@ export const TextInput = forwardRef(
             variant="body"
             color={isValid ? 'text' : 'error'}
             onLayout={measureLabel}
+            numberOfLines={1}
           >
             {label}
             {isRequired && showRequiredAsterisk ? '*' : ''}
           </Text>
         </Label>
 
+        {showCharacterLimit && isFocused && (
+          <CharacterCount style={labelStyles} variant="caption">
```

Comment:
Why does this reuse the `labelStyles`? 🤔 What part of this is animated?

Reply:
The character count element is sharing the same animation as the label

https://github.com/TaitoUnited/react-native-template/assets/45203192/4a889e68-8fd6-4264-9d4e-4de8ee0d41b9